charms/istio-pilot/tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
Please see my above comments regarding unittest vs integration tests.

Reply:
Moved the tests into a /unit directory